client/state/sites/reducer.js

```diff
@@ -53,6 +54,7 @@ export function items( state = null, action ) {
 	}
 	switch ( action.type ) {
 		case WORDADS_SITE_APPROVE_REQUEST_SUCCESS:
+			// eslint-disable-next-line no-case-declarations
```

Comment:
I know this isn't really in the scope of this PR as such, but we can better fix the lint issues by adding curlies around the case block. Doing so would scope the declarations properly:

```js
case WORDADS_SITE_APPROVE_REQUEST_SUCCESS: {
  const prevSite = state[ action.siteId ]; // now scoped only to the case block, not lexically
  ...
}